spencer_caldwell/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "spencer_caldwell",
+  "version": "1.0.0",
+  "description": "Vanilla HTTP Server =========================== To complete this assignment:   * fork this repository (the sub module for this specific assignment)   * clone down your fork   * place all of your work in a folder that is your full name, use `_`s instead of spaces   * push back up to your fork   * create a pull request back to the original repo   * submit a link to the PR in canvas",
```

Comment:
This is the readme, it should go in a readme, or another .txt document called "assignment" or something, not in the description field.